python/defaults/Commissioning18.py

```diff
@@ -0,0 +1,18 @@
+common = {
+	'eras' : ['Run2_2018'],
+	'miniAOD' : True,
+	'storeDeepFlavourTagVariables' : True,
+}
+
+mc = {
+        #'inputFiles' : ['/store/relval/CMSSW_10_1_0_pre2/RelValTTbar_13/MINIAODSIM/100X_upgrade2018_realistic_v11-v1/20000/14AC524C-981E-E811-B6D1-0CC47A4D7632.root'],
+        'inputFiles' : ['/store/mc/RunIISpring18MiniAOD/TTToHadronic_TuneCP5_13TeV-powheg-pythia8/MINIAODSIM/100X_upgrade2018_realistic_v10-v1/30000/008B1124-0B31-E811-BD14-0025904C7A5A.root'],
```

Comment:
This should be some 10.1.X relVal file, have a look at https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatAlgos/python/patInputFiles_cff.py

Reply:
yeah, that works. please update and just push the new commit to the same branch. Git will do its magic and append it to this PR